src/checks/stylelint.ts

```diff
@@ -0,0 +1,40 @@
+import { execaCommand } from 'execa';
+
+export const title = 'Stylelint';
+
+export default async function stylelintCheck() {
+  try {
+    await execaCommand(
+      'pnpm stylelint **/*.{css,scss,less,js,tsx} --max-warnings 0',
+    );
+  } catch (error) {
+    const { stdout } = error as { stdout: string };
+    const lines = stdout.split('\n');
+
+    // If no Stylelint problems detected, throw the error
+    if (
+      !/^\d+ problems? \(\d+ errors?, \d+ warnings\)$/.test(
```

Comment:
what if there are only errors or only warnings? the ESLint check is intentionally more permissive of a check to avoid this problem

Reply:
https://github.com/upleveled/preflight/pull/377/commits/f0dbdb404c6b73fdfdc25cc8870d21dad9c7fef7